src/helpers/utils.ts

```diff
@@ -103,23 +136,41 @@ export function runCommand(cmd: string, requiredRepoType = RepoType.Undefined) {
   } else if (getRepoType() !== requiredRepoType) {
     vscode.window.showErrorMessage(`Command works only with ${requiredRepoType} projects!`);
   } else {
-    const project = resolve(workspace.uri.fsPath, 'package.json');
-
-    try {
-      const { scripts = {} } = __non_webpack_require__(project) || {};
-      const candidates = Object.keys(scripts).filter((m) => scripts[m].trim().startsWith(cmd));
-      const shellCommand =
-        candidates.length === 0 ? cmd : candidates.length === 1 ? scripts[candidates.pop() ?? ''] : undefined;
-
-      if (shellCommand !== undefined) {
-        execCommand(shellCommand);
-      } else {
-        vscode.window.showQuickPick(candidates).then(execCommand);
+    const nodeModulesAreAvailable = existsSync(`${workspace.uri.fsPath}/node_modules`);
+
+    if (!nodeModulesAreAvailable) {
+      // ask user to install node-modules
+      vscode.window
+        .showInformationMessage(
+          'Dependencies are not installed yet, should we install the dependencies now?',
+          'Yes',
+          'No',
+        )
+        .then((answer) => {
+          if (answer === 'Yes') {
+            installDependencies(workspace.uri.fsPath);
+          }
+        });
+    } else {
+      // execute the command - cmd
+      const project = resolve(workspace.uri.fsPath, 'package.json');
+
+      try {
+        const { scripts = {} } = __non_webpack_require__(project) || {};
+        const candidates = Object.keys(scripts).filter((m) => scripts[m].trim().startsWith(cmd));
+        const shellCommand =
+          candidates.length === 0 ? cmd : candidates.length === 1 ? scripts[candidates.pop() ?? ''] : undefined;
+
+        if (shellCommand !== undefined) {
+          execCommand(shellCommand);
+        } else {
+          vscode.window.showQuickPick(candidates).then(execCommand);
+        }
+      } catch (err) {
+        vscode.window.showErrorMessage(
+          `Could not load the "package.json". Make sure the works pace is valid "${project}".`,
```

Comment:
It's "workspace" not "works pace".

Also make sure to load the package.json before doing any of the checks. If we cannot load the package.json then we cannot run a command anyway (because then there won't be node_modules, but we cannot also install modules).